lgc/include/lgc/state/PipelineState.h

```diff
@@ -446,6 +452,8 @@ class PipelineState final : public Pipeline {
   std::unique_ptr<InterfaceData> m_interfaceData[ShaderStageCompute + 1] = {}; // Per-shader InterfaceData
   PalMetadata *m_palMetadata = nullptr;                                        // PAL metadata object
   unsigned m_waveSize[ShaderStageCountInternal] = {};                          // Per-shader wave size
+  bool m_inOutPackStates[ShaderStageGfxCount][2] = {
```

Comment:
I think this is somewhat verbose, we just need `bool m_inOutPackStates[ShaderStageGfxCount][2] = {};`